backend/app.ts

```diff
@@ -0,0 +1,76 @@
+import * as dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({ path: path.resolve(__dirname, "../.env") });
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import session from 'express-session';
+import passport from 'passport';
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
```

Comment:
Do you think we can keep this as an `import` so we are staying consistent with the methodolgy?